diff -urN authselect-1.5.0/profiles/local/password-auth authselect-1.5.0-patched/profiles/local/password-auth
--- authselect-1.5.0/profiles/local/password-auth	2024-01-18 16:54:40.000000000 +0100
+++ authselect-1.5.0-patched/profiles/local/password-auth	2024-07-16 22:22:51.000000000 +0200
@@ -1,3 +1,4 @@
+auth        optional                                     pam_mount.so                                           {include if "with-pam-mount"}
 auth        required                                     pam_env.so
 auth        required                                     pam_faildelay.so delay=2000000
 auth        required                                     pam_faillock.so preauth silent                         {include if "with-faillock"}
@@ -20,9 +21,11 @@
 password    requisite                                    pam_pwhistory.so use_authtok                           {include if "with-pwhistory"}
 password    sufficient                                   pam_unix.so yescrypt shadow {if not "without-nullok":nullok} use_authtok
 password    required                                     pam_deny.so
+password    optional                                     pam_mount.so disable_interactive                       {include if "with-pam-mount"}
 
 session     optional                                     pam_keyinit.so revoke
 session     required                                     pam_limits.so
+session     optional                                     pam_mount.so                                          {include if "with-pam-mount"}
 session     optional                                     pam_ecryptfs.so unwrap                                {include if "with-ecryptfs"}
 session     optional                                     pam_systemd_home.so                                   {include if "with-systemd-homed"}
 -session    optional                                     pam_systemd.so
diff -urN authselect-1.5.0/profiles/local/README authselect-1.5.0-patched/profiles/local/README
--- authselect-1.5.0/profiles/local/README	2024-01-18 16:54:40.000000000 +0100
+++ authselect-1.5.0-patched/profiles/local/README	2024-07-16 22:25:08.000000000 +0200
@@ -1,5 +1,5 @@
-Local users only
-================
+Local users only with extra options
+===================================
 
 Selecting this profile will enable local files as the source of identity
 and authentication providers.
@@ -15,6 +15,9 @@
     Enable automatic creation of home directories for users on their
     first login.
 
+with-pam-mount::
+    Enable pam_mount support to open LUKS encrypted home directories
+
 with-ecryptfs::
     Enable automatic per-user ecryptfs.
 
@@ -67,9 +70,9 @@
 EXAMPLES
 --------
 
-* Enable local profile
+* Enable local profile with pam-mount support
 
-  authselect select local
+  authselect select local --with-pam-mount
 
 SEE ALSO
 --------
diff -urN authselect-1.5.0/profiles/local/REQUIREMENTS authselect-1.5.0-patched/profiles/local/REQUIREMENTS
--- authselect-1.5.0/profiles/local/REQUIREMENTS	2024-01-18 16:54:40.000000000 +0100
+++ authselect-1.5.0-patched/profiles/local/REQUIREMENTS	2024-07-16 22:20:36.000000000 +0200
@@ -1,6 +1,10 @@
 - with-mkhomedir is selected, make sure pam_oddjob_mkhomedir module                       {include if "with-mkhomedir"}
   is present and oddjobd service is enabled and active                                    {include if "with-mkhomedir"}
   - systemctl enable --now oddjobd.service                                                {include if "with-mkhomedir"}
+                                                                                          {include if "with-pam-mount"}
+- with-pam-mount is selected, make sure pam_mount is installed,                           {include if "with-pam-mount"}
+  LUKS encrypted home partition is created                                                {include if "with-pam-mount"}
+  and configured in /etc/security/pam_mount.conf.xml                                      {include if "with-pam-mount"}
                                                                                           {include if "with-altfiles"}
 - with-altfiles is selected, make sure nss_altfiles module is present                     {include if "with-altfiles"}
                                                                                           {include if "with-libvirt"}
@@ -18,4 +22,4 @@
   - users can then configure keys using the pamu2fcfg tool                                {include if "with-pam-u2f"}
                                                                                           {include if "with-pam-u2f-2fa"}
 - with-pam-u2f-2fa is selected, make sure that the pam u2f module is installed            {include if "with-pam-u2f-2fa"}
-  - users can then configure keys using the pamu2fcfg tool                                {include if "with-pam-u2f-2fa"}
\ Kein Zeilenumbruch am Dateiende.
+  - users can then configure keys using the pamu2fcfg tool                                {include if "with-pam-u2f-2fa"}
